tsl/test/CMakeLists.txt

```diff
@@ -59,7 +59,21 @@ if(PG_REGRESS)
     ${PG_REGRESS_OPTS_LOCAL_INSTANCE}
     USES_TERMINAL)
 
+  add_custom_target(provecheck
```

Comment:
Not really sure if `prove` is a good name for tests, unless it is sort of standard to name tap tests this way. I would rather have something simple like `tap` or more specific name to mark tests being designed for multi-instance testing, etc

Reply:
Community PG uses `prove`, so for uniforming, I am sticking with i